torch/jit/frontend.py

```diff
@@ -289,8 +289,14 @@ def _forward(self):
             # Replace potentially unsupported type annotations by "Any"
             arg.annotation = unused_def.args.args[0].annotation
 
-    return build_def(ctx, fn_def, type_line, def_name, self_name=self_name)
+    # If MonkeyType is installed, get all the consolidated type traces
+    # for the arguments from type_trace_db
+    type_trace_db = torch.jit._script._get_type_trace_db()
+    args_and_types = None  # type: ignore[name-defined]
```

Comment:
Try removing all the `#type: ignore`, I think we shouldn't need them now after localizing `args_and_types`